index.js

```diff
@@ -56,7 +56,7 @@ MPU6050.DEFAULT_ADDRESS = MPU6050.ADDRESS_AD0_LOW;
  * the clock source to use the X Gyro for reference, which is slightly better than
  * the default internal clock source.
  */
-MPU6050.prototype.initialize = function(callback) {
```

Comment:
I think it's good to keep it we don't know what the future will be...

I used it in the example shared at:

https://github.com/TizenTeam/mpu6050/tree/sandbox/rzr/master